### ISSUE TITLE
task1

### ISSUE THREAD

**Issue by AShepelevv** [mipt-diht/mipt-latex-fall2018-magnitov#2]
Задание нужно было коммитить в новую ветку task1, а не в master. Чтобы потом открыть Pull Request и там вести обсуждения по поводу задания.

**Comment by AShepelevv** [comment]
Извиняюсь, не увидел, что ты одобрил PR. Так делать не нужно, я сам смержу, когда больше не будет вопросов. Если бы не заметил, бал бы не поставил. В принципе изменения приняты, все хорошо.

**Comment by magnitov** [comment]
А когда я обновляю файлы, на которые уже открыт PR, тебе приходят уведомления об этом?
Я просто никогда не пользовался гитом раньше, поэтому не знаю как это работает.

**Comment by AShepelevv** [comment]
Да, конечно, когда добавляешь человека (меня) в reviewers или assigners. Поэтому и прошу делать именно так, чтобы ничего не пропустить.
<img width="858" alt="screenshot 2018-10-22 at 15 08 41" src="https://user-images.githubusercontent.com/23651313/47291482-7a9f1e00-d60c-11e8-98d5-47d479cdb801.png">
